File: DKITProject/DAL/Models/Workshop.cs
using System;

namespace DKITProject.DAL.Models
{
    public class Workshop
    {
        public int Id { get; set; }
        public string Headline { get; set; }
        public string Announce { get; set; }
        public string Content { get; set; }
        public string ImgIcon { get; set; }
        public string ImgPreview { get; set; }
        public DateTime DatePost { get; set; }
        public DateTime DateStart { get; set; }
        public DateTime DateEnd { get; set; }
        public int Count { get; set; }
        public WorkshopParticipant[] WorkshopParticipants { get; set; }
    }
}

File: DKITProject/ClientApp/src/app/models/workshop.ts
export class Workshop {
    constructor(
        public id: number,
        public headline: string,
        public announce: string,  
        public content: string,
        public imgIcon: string,
        public imgPreview: string,
        public datePost: Date,
        public dateStart: Date,
        public dateEnd: Date,
        public count: number
    ) {}
}

export class WorkshopPreview {
    constructor(
        public id: number,
        public headline: string,
        public announce: string,
        public imgIcon: string,
        public datePost: Date,
        public dateStart: Date,
        public dateEnd: Date,
        public count: number
    ){}
}